### DISCUSSION TITLE
Is Analytics supposed to work with pages router?

### DISCUSSION THREAD

**Discussion by johnedwardobrien** [vercel/community#6349] [Help]
### Summary

I'm trying to use Analytics with pages router and _document. SpeedInsights seems to work fine, this is my current configuration in the _document: https://github.com/johnobrien8642/personal-site/blob/master/pages/_document.page.tsx

Should that work? Is there a way I can check in the browser if it's loaded? I'm not seeing any data and haven't since I tried using it a couple months ago. Sidelined my concern for the time being but now I'd like Analytics.

### Example

https://github.com/johnobrien8642/personal-site/blob/master/pages/_document.page.tsx

### Steps to Reproduce

1. Have a next.js project using pages router (my ver is 13.14.12)
2. Try this configuration from my _document: https://github.com/johnobrien8642/personal-site/blob/master/pages/_document.page.tsx Specifically the placement of Analytics within the body tag.
3. Deploy and go to your deployments analytics, open your deployed page, but observe no analytics.

**Comment by versecafe** [comment]
@johnobrien8642 your repository does not appear to be public so I can't do much to help, but Vercel Analytics should work in the pages router.
